SecurityPkg/Library/Tcg2InitEventLib/Tcg2InitEventLib.c

```diff
@@ -0,0 +1,143 @@
+/** @file
+  Initializes TCG event data structures
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <Library/Tpm2CommandLib.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseLib.h>
+#include <Library/UefiLib.h>
+#include <Library/UefiLib/UefiLibInternal.h>
+#include <Library/UefiBootServicesTableLib.h>

```

Comment:
Is `UefiBootServicesTableLib` actually needed?

Reply:
Please double check others too. A `BASE` library shouldn't have a dependency on something like boot services.